tests/dummy/app/components/less-boilerplate/component.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+import hbs from 'htmlbars-inline-precompile';
+import connect from 'ember-redux/components/connect';
+import { bumpTwice } from '../../actions/index';
+
+var stateToComputed = function(state) {
+  return {
+    low: state.low
+  };
+};
+
+var mapDispatchToActions = {
+  bumpTwice
+};
+
+var LessBoilerplateComponent = Ember.Component.extend({
+  actions: {
+    handleClick() {
+      this.actions['bumpTwice'](1);
+    }
+  },
+  layout: hbs`
+    <span class="upp-low">{{low}}</span>
+    <button class="btn-upp" onclick={{action "handleClick"}}>upp</button>
```

Comment:
Just curious, could we add another button here that actually passes in the initial value? As long as you don't need the actual jQuery event, won't this be a solution too; 

```hbs
<button onclick={{action "bumpTwice" 1}}>upp with value</button>
```

Unless I've completely forgotten something about the way actions curry their arguments in Ember though?